# HG changeset patch
# Parent  b4a8bb3a5960a96540f6f925fdeac4a32fdcc114
Document the unix:// scheme for lightsc.py's -u option

diff --git a/docs/first-steps.rst b/docs/first-steps.rst
--- a/docs/first-steps.rst
+++ b/docs/first-steps.rst
@@ -259,13 +259,19 @@
 
 Check out :doc:`lightsd's protocol </protocol>` to see everything you can do.
 
-lightsc.py also accepts an url which lets you connect to any running lightsd,
-e.g:
+lightsc.py also accepts an url which lets you connect to anything running
+lightsd, e.g:
 
 ::
 
    lightsc.py -u tcp://localhost:1234
 
+Or, for an Unix socket:
+
+::
+
+    lightsc.py -u unix:///path/to/lightsd/socket
+
 .. _lightsc.py: https://github.com/lopter/lightsd/blob/master/examples/lightsc.py
 
 .. vim: set tw=80 spelllang=en spell:
